recipes/nds2-client-swig/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "nds2-client-swig" %}
+{% set version = "0.16.0" %}
+{% set sha256 = "db785868f519a07e3a1cf7216cf5610a984d96e3795862794de98215534aa463" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.xz
+  url: http://software.ligo.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - make
+    - cmake >=3.6
+    - pkg-config
+    - swig >=3.0.7
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - nds2-client ={{ version }}
+    - python
+    - numpy {{ numpy }}
+
+test:
+  imports:
+    - nds2
+
+outputs:
+  - name: python-nds2-client
+    requirements:
+      host:
+        - nds2-client ={{ version }}
+        - python
+        - numpy
+      run:
+        - nds2-client ={{ version }}
+        - python
+        - {{ pin_compatible('numpy') }}
+    files:
+      - lib/python*/site-packages/nds2.py*
+      - lib/python*/site-packages/__pycache__/nds2.*.pyc  # [not py27]
+      - lib/python*/site-packages/_nds2.so
+    test:
+      imports:
+        - nds2
+    about:
+      home: https://wiki.ligo.org/DASWG/NDSClient
+      license: GPLv3
+      license_family: GPL
+      summary: Python extensions for NDS2
+      description: |
+        This provides python wrappers for the nds2 client.
+
+about:
+  home: https://wiki.ligo.org/DASWG/NDSClient
+  license: GPLv3
+  license_family: GPL
```

Comment:
Since this is GPL, you need to provide the `license_file` key here too

Reply:
@scopatz, the tarball for this package does not include a license file. Should I create a patch that includes it, or can this be omitted?

Reply:
No, since it is GPL'd you can't ommit it sadly.  However, you can include a copy of the license in the recipe: https://conda-forge.org/docs/meta.html#packaging-the-license-manually

Reply:
ok, I've added that in 6595d2695b73334462e9efcce1148c2dbca58be7.